frontend/app/src/components/inputs/AssetSelect.vue

```diff
@@ -267,6 +267,9 @@ const listeners = useListeners();
         />
       </div>
     </template>
+    <template #prepend>
+      <slot name="prepend" />
+    </template>
```

Comment:
![image](https://github.com/rotki/rotki/assets/26648140/e3c6f765-59ec-4ac2-8704-0054ea161290)

`#prepend-inner` looks better I think (?)

Reply:
It does but it also pushes the label in the asset select itself when there is a selection so using it is a bit more complicated

Reply:
We can do it in a new PR at some point, but I didn't want to mess with the AssetSelect css 😢